sqlmesh/magics.py

```diff
@@ -598,6 +598,12 @@ def create_external_models(self, context: Context, line: str) -> None:
         action="store_true",
         help="Show a sample of the rows that differ. With many columns, the output can be very wide.",
     )
+    @argument(
+        "--decimals",
+        type=int,
+        default=20,
```

Comment:
3 not 20 right

Reply:
ah, thanks for catching this. Not sure how this happened.